### ISSUE TITLE
ERROR! File not found (contigs) running on biocontainers/quast:5.2.0

### ISSUE THREAD

**Issue by Vivian-chen16** [ablab/quast#247]
Hello,

I was trying to use Nextflow building the pipeline and having Quast for assembly QC.  However, I kept encountering `ERROR! File not found (contigs)` issue while it worked well pulling image itself to run my test sample. I believe all the dependencies have been taken care of within the image (but correct me if I'm wrong). Appreciate any suggestion for troubleshooting!

Below are some details for reproducing if needes:
- Ubuntu 22.04.2 LTS
- nextflow version 23.04.2.5870
- biocontainers/quast:5.2.0--py39pl5321h2add14b_1
- [sample data](https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000292685.1/)
- [.command.log](https://github.com/ablab/quast/files/11969695/default.command.log)

**Comment by alexeigurevich** [comment]
This error means that you simply didn't specify contigs (genome assembly) with your Quast command, this is the only required parameter for Quast. E.g., if your assembly file is `assembly.fasta`, you can run Quast simply as
```quast.py assembly.fasta``` 
(if you are in the directory containing this file, otherwise please specify the full path to it). 

Note that normally, `quast.py` without any parameters should print the help message with available options but I was able to reproduce the error by specifying command-line Quast an empty path:
`quast.py ''`

To sum up, this seems to be not an issue of Quast but rather a problem with how you run it :) 
Or with how it is implemented in biocontainers, we are not aware of this wrapper for our tool. However, there should be a way to provide a path to input file for the wrapped tool.

**Comment by Vivian-chen16** [comment]
Hi @alexeigurevich,

thanks for the quick response! I double-checked the command I ran and the `.fasta` seems presented (please see the block below from `.command.sh` for details). If there's any missing part, please shed a light (I've been blocked on this for a while :/ ). Thanks a lot!!

```
#!/bin/bash -euo pipefail
quast.py \
    P566H11b.fasta \
    --gene-finding \
    --output-dir SAMPLE_1_T1 \
    --threads 6 \
    --min-contig 500 \ 


ln -s SAMPLE_1_T1/report.tsv

cat <<-END_VERSIONS > versions.yml
"NFCORE_ASSEMBLYQC:ASSEMBLYQC:QUAST":
    quast: $(quast.py --version 2>&1 | sed 's/^.*QUAST v//; s/ .*$//')
END_VERSIONS
```

**Comment by asl** [comment]
You are having extra newline after `--min-contig` hence the empty argument
